conda-recipe/meta.yaml

```diff
@@ -43,6 +23,40 @@ test:
     - pytest <7.2.0
     - pytest-qt
 
+outputs:
+  - name: lightpath-base
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - setuptools_scm
+      run:
+        - python >=3.6
+        - coloredlogs
+        - happi >=1.6.0
+        - numpy
+        - ophyd
+        - prettytable
+
+  - name: lightpath
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python >=3.6
+        - pip
+        - setuptools_scm
+      run:
+        - lightpath-base
```

Comment:
I think the weird pin invocation from the other recipes is required to make sure lightpath x.x.x depends on lightpath-base at exactly x.x.x

Reply:
I don't think this is why the CI is having an issue

Reply:
Adding it did move us forward.  This seems to build fine for me locally so I'll have to abuse GHA a bit here debugging